src/widget/wmainmenubar.cpp

```diff
@@ -156,8 +156,8 @@ void WMainMenuBar::initialize() {
     pLibraryMenu->addAction(pLibraryRescan);
 
 #ifdef __ENGINEPRIME__
-    QString exportTitle = tr("E&xport Library to Engine Prime");
-    QString exportText = tr("Export the library to the Engine Prime format");
+    QString exportTitle = tr("E&xport Library to Engine DJ");
+    QString exportText = tr("Export the library to the Engine DJ format");
```

Comment:
Let's add a comment for translators that 'Engine DJ'  is a brand / name and should not be translated.
The syntax is `//: hint text` and that goes one the line above the tr strings, so here it'd be
`//: "Engine DJ" must not be translated`